service/arangod_config_builder.go

```diff
@@ -45,6 +45,19 @@ import (
 	"github.com/rs/zerolog"
 )
 
+var (
+	urlFixer = strings.NewReplacer(
+		"http://", "tcp://",
+		"https://", "ssl://",
+	)
+)
+
+// FixupEndpointURLScheme changes endpoint URL schemes used by arangod to ones used by arangodb.
```

Comment:
'FixupEndpointURLScheme changes endpoint URL schemes used by arangod to ones used by arangodb.' -> 'FixupEndpointURLScheme changes endpoint URL schemes used by Starter to ones used by arangod.'

Reply:
Fixed